packages/pyright-internal/src/analyzer/sourceFile.ts

```diff
@@ -80,7 +83,74 @@ export enum IPythonMode {
     CellDocs,
 }
 
+class WriteableData {
```

Comment:
Made a class to hold all changeable data. Maybe I should call it MutableData. 

Anyway, this was a lot simpler than copying each item out by hand and felt like it would be more obvious in the future what needs to go here. 

However, it made a lot of changes in this file because of it.